packages/framework/src/Component/FileUpload/ImageUploadData.php

```diff
@@ -12,7 +12,7 @@ class ImageUploadData
     /**
      * @var string[]
      */
-    public $uploadedFileNames = [];
```

Comment:
Do you thing `$filenames` is better naming than `$fileNames `? I'm quite against it.

Reply:
Yes